static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,70 @@
+# Import Data
+
+We've seen how to [push](/doc/get-started/store-data) and
+[pull](/doc/get-started/retrieve-data) data a <abbr>DVC project</abbr>'s
+[remote](/doc/commands-reference/remote). But what if we wanted to integrate a
+dataset or ML model produced in one project into another project? (As an
+[external dependency](/doc/user-guide/external-dependencies))
+
+One way is to download the data (with `wget` or `dvc get`, for example) and use
+`dvc add` to track it locally, but the connection between projects would be
```

Comment:
locally -> within the project?

Reply:
Which project? There are 2 in question. Changing to just "track it".